web/modules/custom/bglobal_angular/components/voices_spotlight/voices_spotlight.info.yml

```diff
@@ -0,0 +1,29 @@
+name: 'Angular: Voices Post Spotlight'
+machine_name: voices-spotlight
+type: pdb
+description: 'Voices Spotlight angular template'
+package: NG2
+version: '1.0.0'
+core_version_requirement: ^8 || ^9
+module_status: active
+presentation: ng2
+configuration:
+  info:
+    type: item
+    title: 'Data endpoint. Defaults to most recent featured post.'
+    description: 'https://voices.berkeley.edu/single-post/%/tag/%/cert/%/area/International'
```

Comment:
We are using `{$post}` instead of `%` in other documentation.

Reply:
The use case for this component is only one post. For more posts we have the Voices List one.